ci/quickstart-bitbucket-master.json

```diff
@@ -31,6 +31,10 @@
         "ParameterKey": "DBPassword",
         "ParameterValue": "$[taskcat_genpass_8S]"
     },
+    {
+        "ParameterKey": "ElasticSearchPassword",
```

Comment:
Is there one test testing the optionality of the `ElasticSearchPassword` parameter? Even though it is not recommended, it would be good to have a test for it.

Reply:
Good idea